src/main/java/pl/shop/models/Customer.java

```diff
@@ -12,8 +12,13 @@
 import java.util.Set;
 import java.util.UUID;
 
+@AllArgsConstructor
+@NoArgsConstructor
 @Entity
 @Data
+@Getter
+@Setter
+@Table(name = "customers")
```

Comment:
Czy to potrzebna zmiana ? Czy w tym przypadku liczba pojedyncza nie wyglądałaby lepiej ?

Reply:
> Czy to potrzebna zmiana ? Czy w tym przypadku liczba pojedyncza nie wyglądałaby lepiej ?

Wedle uznania, dla mnie czytelniejsza jest liczba mnoga. Ogólnie tabela to zbiór, w tym przypadku klientów, a nie klienta :D. Poza tym w encji Order, apka się wysypywała, bo 'order' to słowo kluczowe w sql i jest zabronione.

Reply:
Możemy przyjąć taką konwencję :)